### DISCUSSION TITLE
Discord server should allow GIFs

### DISCUSSION THREAD

**Discussion by adityaraute** [EddieHubCommunity/support#671] [Ideas] [answered]
I know that only GitHub links are allowed in the chat, from members who do not have a specific role. I understand the motive is to prevent spam. 

However, to preserve the fun and craziness of the chat, and to build better relationships, the server should allow the Tenor GIFs that Discord provides. A lot of people tend to express themselves better through these than emojis, and all this links are safe. Hence, I propose that the server allows people to post Tenor links.

If enough people agree to my proposal, I shall create an issue for the same. If one is already created, please link it below.

**Comment by Vyvy-vi** [comment, marked as answer]
I think gifs are already allowed...
(there was an issue for this earlier: [support#580](https://github.com/EddieJaoudeCommunity/support/issues/580))

**Comment by adityaraute** [comment]
That was foolish of me to not test it well before.

**Reply by eddiejaoude** [reply]
We all do it 😂
